internal/runners/manifest/requirements.go

```diff
@@ -66,9 +66,13 @@ func (o requirements) MarshalOutput(_ output.Format) interface{} {
 
 func (o requirements) MarshalStructured(f output.Format) interface{} {
 	for _, req := range o.Requirements {
+		if req == nil {
+			continue
+		}
```

Comment:
This feels like it should happen earlier, ie. when `o.Requirements` is calculated. Unless there's a reason for having empty entries?